src/cirrus/lambda_functions/update_state.py

```diff
@@ -9,19 +9,12 @@
 from cirrus.lib.events import WorkflowEventManager
 from cirrus.lib.logging import get_task_logger
 from cirrus.lib.process_payload import ProcessPayload
-from cirrus.lib.utils import SNSPublisher, SQSPublisher, cold_start, get_client
+from cirrus.lib.utils import SNSPublisher, SQSPublisher, cold_start
 
 cold_start()
 
 logger = get_task_logger("function.update-state", payload=())
 
-# boto3 clients
-SFN_CLIENT = get_client("stepfunctions")
```

Comment:
We'll want to make a ticket on the terraform side to remove the IAM policy grant for access to step functions actions.